Cargo.toml

```diff
@@ -28,10 +28,13 @@ std-fs-io = ["casper-types/std-fs-io"]
 [dependencies]
 async-trait = { version = "0.1.74", optional = true }
 base16 = "0.2.1"
+base64 = "0.21.5"
+bytes = "1.5.0"
 casper-hashing = "3.0.0"
 casper-types = { version = "4.0.1", features = ["std"] }
 clap = { version = "~4.4", optional = true, features = ["cargo", "deprecated", "wrap_help"] }
 clap_complete = { version = "<4.5.0", optional = true }
+flate2 = "1.0.28"
```

Comment:
Is it possible to remove the default feature and still work ? thanks

Reply:
Unfortunately, not.